intake/tests/test_top_level.py

```diff
@@ -47,11 +56,18 @@ def test_user_catalog(user_catalog):
     assert set(cat) >= set(['ex1', 'ex2'])
 
 
+def test_path_catalog(tmp_path_catalog):
+    intake.config.conf['catalog_path'] = os.path.join(tempfile.gettempdir(), 'intake')
+    cat = intake.load_combo_catalog()
+    time.sleep(2) # wait 2 seconds for catalog to refresh
```

Comment:
This shouldn't be necessary, it refreshes on instantiation.

Reply:
Ah ok! I was just copying the other test.